### ISSUE TITLE
Replacing current security .45 pistols with more suiting weaponary

### ISSUE THREAD

**Issue by saysage** [VOREStation/VOREStation#2649]
In my opinion, current solid-projectile sidearm for security is making very little sense both logical-, historical- and code-wise, as it's being M1911 (7-shot .45 handgun) with sprites of Glock 17. Why this makes little sense, is because of few moments:
1) For the handgun issued to the members of security of one of the largest megacorporations in current 26th century, it's just way too odd to use parameters of one of the earliest ever semiautomatic handgun designs, which M1911 is. While gun itself is legendary for reliability and power, it's making very little sense for officers of far away future to have it on the same racks as high-tech pain-inducing tasers and electrosphere-sending stunrevolvers. Yes, something like this would suit other, more retrofuturistic in atmosphere server, but V.O.R.E. always went more sci-fi way with things like resleeving, NIFs and many other. To mention retro weapons, ones like detective's revolver and bartender's shotgun won't ever raise any questions as those weapons aren't intended for specific combat purposes, while tasers, stun-revolvers and .45 pistols are issued to officers as standart everyday gear.
2) Compared to other two guns, taser and stun-revolver, solid-projectile weapon always is in very high disadvantage. Main and almost always only reasons for officers to take those over taser and stun-revolver - ability to harm mobs like solar grubs and invulnerability to EMP powerlosses, and ability to use different ammo. But both three situations take place very rarely, especially ammo - officers are by standart issued only with flash ammo, warden can legally give by request rubber ammo, and lethal one is almost never used as it's legally given out only in case of actual danger for crew and station - and exactly in such situations pistols aren't used, as they are changed by lasers and shotguns, which makes all three advantages pretty questionable, as here go all the disadvantages. As well as stun-revolver's spheres, solid projectiles can't go through glass, but instead of just blowing in sparks those damage windows, making it more possible to create emergency by accident. Tasers don't have such problem at all, and stun-revolvers are powerful enough to lay enemy on the floor with one or two shots only, while handgun does not have such profits. Also, recoil, which is not present in both energy weapons, makes .45 pistol much harder to shoot it precisely, which is second large disadvantage. So, when taser is precise, can hit through windows and has large capacity, stun-revolver is dramatically strong and can leave enemy stunned with just one shot, solid-projectile resprited-renamed M1911 is only good because it's not suspectible to EMP, can use various ammo and hurts mobs slightly. That't not very much, actually?

My suggestion how to fix this is to turn those parodies on handguns into either actual Glocks 17 instead of M1911, as Glocks are popular world-wide for thirty years now in both army and police forces, compact, reliable, high-capacity 9mm sidearm, which would be really easy to add as there already are sprites for it and those are literally the ones used for crappy .45 pistol. 9mm weapons deal less damage than .45 (20 instead of 25 with lethal ammo), making chances of accidental hull breach or murdering someone lower, and magazine with 17 bullets in it will compensate low bullet effect and inability to shoot through glass.
Or, it can be replaced with completely new, suiting futuristic 26th century handgun, which I can myself draw as I'm pretty skilled in creating SS13 sprites. In terms of balance, using stats of Glock for it would be good as well, it just should be something what can be issued to security forces with proud face, not embarassed one.

**Comment by Yoshax** [comment]
So. Let's get things straight for a moment. You want to replace the 7 shot 25 damage gun, with a 17 shot 20 damage gun?

.45: 7 * 25 = 175
Proposed 9mm: 17 * 20 = 340
Revolver: 6 * 60 = 360
Machine Pistol; 20 * 20 = 400
DMR: 10 * 35 = 350 (15 armor penetration)

So, just to get things straight, and cut through all the shit here, you just want to buff security?

If you want a 17 shot "Glock" with the same balance, then you get 10 damage. Not 20.

**Comment by saysage** [comment]
Few things to mention, in development channel it was discussed that ammo damage depends on stuff like ammo type, not exactly caliber, and currently lowest pistol one gives out 20 damage. I'll say that 15 would be much better but 10 would be not, personally thought until this day that 9mm dealt 15. Such measurements as you did do not count things as how quick this damage can be dealt - for sure, gun which deals 10 damage in 10 shots and gun which deals 50 in 2 shots are VERY different.
Plus, I think that even from your own results you can see how bad actually this .45 pistol is, Glock with 17 shots and 15 damage in total gives 255 damage, which should be p alright taking in mind that single bullet would deal four times less damage than revolver one.

**Comment by Yoshax** [comment]
It's meant to be bad. It's not meant to stand close to, or match with the revolver, or the DMR, or any of those weapons.

**Comment by saysage** [comment]
But why is it meant to, then? It really feels way too underpowered near to tasers and stun-revolvers in terms of balance, and I described in details why. I see no reason to make it lose so much to other standart-issue weapons, there is no reasoning for specifically "being bad". We are not having action-filled gameplay to specifically cut security's powers in order to make them less threat to antags as usually strict balancing and underpowering needs, but still, all the flaws of it still appear. It's just not suiting. It's okay if this gun is much weaker than revolver, it still will be even if it will be Glock 17 with 17 shots 15 damage each, but it won't be such a complete unfitting nonsense as it is right now.

**Comment by SpadesNeil** [comment]
To be honest I've kind of been mulling over the possibility for a high capacity pistol option too, but Yosh explains exactly why I didn't say anything about it.

Balance is a major problem.

In fairness the current option for handguns is just downright pathetic, but I think we're better off doing a 10 round magazine before we do 15, 17, 20 rounds in a mag.

**Comment by saysage** [comment]
Ace, but not capacity defies balance and logical sense behind weapon, as no matter if your gun has 7, 10 or 17 shots, changing mags takes just few seconds for most people, and just two clicks for experienced ones. With high-power .45 caliber it isn't even needed to spend much ammo. Capacity is something what reaches 15-20 shots even in today designs, while sheer kinetic force of bullet isn't what handgun ammo is usually aimed at. So even 10 shots doesn't make much sense when tiny Glock is filled with seventeen bullets.
To be an actual valuable weapon, solid-projectile ammo handgun should have stats which can be counted fair in comparison to basic taser, which already has advantages of shooting through glass, using beam instead of slow-moing projectile, and is completely dangerless for hull and surroundings. Why not to, as I proposed, change caliber to lower-damage one and raise capacity more dramatically so it will actually be way more balanced weapon? 17 MIGHT be too much (but it won't look so if low-power pistol bullets will have damage lowered to 15 from 20, as right now actual difference between medium (25) and small (20) pistol bullets doesn't feel at all), but 10, for sure, isn't much better option. Especially with such old kind of ammo as .45 ACP, you know. 
What about making completely new handguns, eh? Something actually suiting sci-fi, futuristic setting while also being not a usual energy-based gun? Maybe main mistake is looking at designs we have right now, and not implementing something new. There are many ideas which can have same balance while also not looking pathetic. What about completely original handguns? Glock is something I suggested because this is the first semiauto pistol most people think of when they are asked for popular nowadays 9mm handgun. That's it.
This all is still just my opinion, I'm open to hearing why it's bad and I'm glad to know that Ace understands how bad current guns are. Any changes are for better, true, but some can be better than others. Maybe try making a vote for virgo people on that? I don't know.

**Comment by saysage** [comment]
Also why the heck is revolver so OP then? 60 damage, seriously? Was that about matheba or detective's revolver?

**Comment by SpadesNeil** [comment]
You're still limited in how much ammo you can carry under the same backpack slots. You misunderstand. This is not a matter of damage output per second. It's a matter of total possible damage output for the same inventory space.

**Comment by SpadesNeil** [comment]
And no, 10 round capacity _doesn't_ make much sense.

But anything above 10 rounds is considered "high capacity" according to some IRL jurisdictions. Not that I agree with them. But it sounds like something NT would do.

This also means we could meet a middle ground on this and offer high capacity magazines alongside of the non-high-capacity ones.

**Comment by saysage** [comment]
Eh, 10 is better too if it will get through, despite actually hoping for better. Will it still be .45-ammo weapon or 9mm weapon? Or something new?

**Comment by saysage** [comment]
Ay, how nice. Thank you papa Ace :^)

**Comment by SpadesNeil** [comment]
I had to close the Polaris issue because their policy on issue posting had changed, but for us, we'll keep discussing here.

**Comment by Yoshax** [comment]
If it were up to just me, I'd remove it entirely. There's next to no reason for Security to have ballistics as a standard thing. They serve no use.  Why do they need a ballistic firearm as standard equipment? Other than for the sake of having a gun that shoots bullets because it's fun?

Also, @saysage you may want to invest in some line breaks. It's close to impossible to read most of what you're posting, it's a neverending wall of text.

**Comment by SpadesNeil** [comment]
https://www.youtube.com/watch?v=ZJlF3xcYDw8 Deterrents.

**Comment by Yoshax** [comment]
I do not think there is a single reasonable, feasible way to make any ballistic weapon comparable, or preferably to either the Stun Revolver or Taser in almost every situation. If the situation is serious enough to call for lethal weapons, than the point has been reached where a .45 or 9mm pistol is no longer sufficient. Before that point, a Stun Revolver or Taser are far more effective on almost every target.

**Comment by SpadesNeil** [comment]
And I think it's absurd that we don't equip our Security officers with hand guns by default, forcing them to flee back to the armory to get any weapons at all the moment a threat appears that _does_ warrant lethal force, allowing said threat to continue causing damage for a period of time.

But if we start equipping the Security staff with SMGs I bet people will not appreciate that.

**Comment by Yoshax** [comment]
Can you explain to me, and provide examples, a scenario in which a lethal .45 is going to be more effective than stun revolver?

**Comment by SpadesNeil** [comment]
When you need the target to not get back up. Ever.

Also literally any situation involving someone violent who happens to be on pain inhibiting drugs.

**Comment by Yoshax** [comment]
Okay, I can understand the pain one, sort of. But that... A very minor case. It also highly depends on whether or not the person has injected it, or ingested it via pill or other means. If they took a pill or suchlike then most normal means to incapacitate them will function fine. You could also use a Capture gun to net them, assuming they are not a major threat.

I do not see a situation where you would want to be shooting someone, where another weapon would not be better. If you are facing off against a dangerous threat, let us say that you both have a lethal .45. It's down to pure luck and RNG over who gets out ontop, assuming equal armor. However, in the case of a stun revolver up against a lethal .45 the stun revolver comes out on top in almost every case, because it puts the target down faster. If you, as you say, don't want someone to get back up ever, then the best way to actually do that would be a combination of both weapons, by downing them with the stun revolver and executing them with the .45. 

You bring up the point that nobody wants to be knocked out, but you're wrong. I come from a server where this kind of thing was dealt with every round. I can tell you with certainty that the vast majority of the population, in a situation where lethal weapons are out, would OOCly rather be dead then knocked out. People do not enjoy brig RP. You're assuming that people would rather spend time RPing processing, being brigged, having their injuries treat and surgery done, and whatever else comes with being shot with lethals rather than just being dead.

**Comment by SpadesNeil** [comment]
Respectfully, I feel like you're grasping for straws as to why non lethal weapons should not be a thing. I'm not sure why we're having this argument to begin with. They're already only supplied at the discretion of the HoS and maybe Warden rather than handed out to every security guard.

A good security officer is going to know when to stun and when to kill. That is why we don't give them as standard equipment that any officer can immediately grab, but at the same time why they're also allowed in code green _if_ the current boss thinks a particular officer is good enough to be trusted with a proper firearm. The same security officer is going to also know that his pepper spray won't work very well on someone wearing glasses, or he doesn't need to chase down and stun baton someone who is just marking up the halls with crayon.

Also that last paragraph is outright absurd to me so I'm not really able to respond. I've met _no one else_ who agrees with that on any heavy RP server, except people who just want to greytide around with throw-away characters, and they just log out when stunned anyway.

tl;dr the argument to remove lethals is absurd and really isn't a relevant part of this discussion what so ever

**Comment by Yoshax** [comment]
Have you asked anyone else? Because most of the people I've met, and who have committed a crime worthy of having lethals fired at them, would rather be dead than shot and brigged. 

I'm not trying to make a case for non lethal weapons not existing, I'm presenting a case as to why ballistic pistols are fairly pointless. The one effective use of them is flash rounds, and that's because they have fairly good accuracy, confuse the target, and cause good halloss. Rubbers rounds are utterly useless. Lethal rounds are... Medicore, and outclasses by quite literally everything else. This issue is about replacing one pistol with another pistol, but what is actual benefit or point? 

The discussion _should_ be about finding a proper use for them by changing them in some way, or removing them. Because as of right now, changing the calibre, or number of bullets, or sprites, or anything of that sort isn't really going to do anything meaningful.

Even in the case where a lethal weapon is a good idea, and would be given, a lethal energy-based weapon is superior in most cases, They provide the ability to shoot through windows, stronger against most forms of armour and are more humane due to not having the risk of internal organ or bone damage.

**Comment by SpadesNeil** [comment]
https://github.com/PolarisSS13/Polaris/pull/4480 Coming soon to a station near you.
